doc/getting-started.md

```diff
@@ -106,6 +106,43 @@ Using the *shared storage* strategy, you can simulate *dedicated storage* by
 on.
 
 
+Serial 
+-----------------
+
+Many `AArch64` devices do not come with a display or can be used in headless mode (as an example - a small home server which doesn't need a display).
+
+In those cases, using serial to interact with the device can be required. Especially before your operating system of choice is loaded.
+
+Tow-Boot uses the same baud rate for all platforms and devices: `115200` , even platforms which generally default to another serial baud rate, have been configured to use the above for homogeneity.
```

Comment:
```suggestion
Tow-Boot uses the same baud rate for all platforms and devices: `115200`. This applies to all platforms, even those which generally default to another serial baud rate.
```

I find the original a bit long and clumsy. Though I'm open to other suggestions.

Reply:
Thank you for the suggestion. The shorter sentences are noticeably easier to read!